ParisBNF/abb/BNFabb102.xml

```diff
@@ -0,0 +1,863 @@
+
+             <msPart xml:id="p2">
+                  <msIdentifier/>
+                  <msContents>
+                     <summary/>
+
+                     <msItem xml:id="p2_i1">
+                         <locus target="#52v"/>
+                         <title type="incomplete" ref="LIT2799RepCh83"/>
+                         <textLang mainLang="gez"/>
+                         <incipit xml:lang="gez"><hi rend="rubric">ሰላም፡ ለከ፡ ጊዮርጊስ፡<gap reason="ellipsis" resp="DN"/> ዘልዳ፡
+                       መስተፅዕነ፡ ፈረስ፡ ፃዕዳ፡</hi>
+                       </incipit>
+                         </msItem>
+                        </msContents>
+                        <physDesc>
+                           <objectDesc form="Codex">
+                              <supportDesc>
+                                 <support>
+                                    <material key="parchment"/>
+                                 </support>
+                                 <extent>
+                                    <measure unit="leaf">1</measure>
+                                    <locus target="#52"/>
+                                    <note><ref target="#p2">Unit 2</ref> is composed of only
+                                       one single leaf. It was glued upon <locus target="#52"/> of
+                                       <ref target="#p1"/> in which a rectangular opening was cut
+                                       (to let the text on the verso-side be visible).</note>
+                                 </extent>
+
+                              </supportDesc>
+
+                              <layoutDesc>
+                                 <layout columns="1" writtenLines="16">
+                                      <locus target="#52"/>
+                                      <note corresp="#textarea2">The original size of the leaf is unknown
+                                        as its margins were cut,
+                                        probably comparable to the other leaves of the volume.</note>
+                                 </layout>
+                              </layoutDesc>
+                           </objectDesc>
+                           <handDesc>
+             <handNote xml:id="h3" script="Ethiopic" corresp="#p2">
+             <seg type="script">18th or 19th century.</seg>
+            <seg type="ink">Black, red.</seg>
+            <desc>Somewhat careless, irregular; by a mediocre scribe.
+            </desc>
+            </handNote>
+            </handDesc>
+                           <decoDesc>
+                             <decoNote xml:id="d61" type="miniature">
+                                 <locus target="#52r"/>
+                                 <desc><ref type="authFile"
+                                   corresp="AT1031VirginAndChild">
+                                   The Virgin and Child, the prone figure of the owner
+                                    depicted beneath</ref>.</desc>
```

Comment:
I would not use note here, but write the entire description inside desc

Reply:
done